lib/resolvers/project.js

```diff
@@ -103,5 +103,19 @@ module.exports = ({ models }) => ({ action, data, id, meta }, transaction) => {
       .then(() => Project.queryWithDeleted(transaction).findById(id));
   }
 
+  if (action === 'delete-amendments') {
+    return ProjectVersion.query(transaction).where({ projectId: id })
+      .then(versions => {
+        const granted = versions.find(v => v.status === 'granted');
```

Comment:
This only finds _a_ granted version. How do we know it's the _most recent_ granted version?

Reply:
Lenny makes a good point - we'd need an orderBy